### ISSUE TITLE
Coveralls badge is detected incorrectly

### ISSUE THREAD

**Issue by mlms13** [npms-io/npms-analyzer#238]
Given the following markdown in the README:

```md
[![test coverage](https://img.shields.io/coveralls/github/mlms13/bs-decode.svg?style=flat-square)](https://coveralls.io/github/mlms13/bs-decode)
```

npms is producing the following metadata for the badge:

```json
{
  "urls": {
    "original": "https://img.shields.io/coveralls/github/mlms13/bs-decode.svg?style=flat-square",
    "service": "https://coveralls.io/repos/github/mlms13/badge.svg?branch=bs-decode",
    "shields": "https://img.shields.io/coveralls/github/mlms13/bs-decode.svg",
    "content": "https://img.shields.io/coveralls/github/mlms13/bs-decode.json"
  },
  "info": {
    "service": "coveralls",
    "type": "coverage",
    "modifiers": {
      "branch": "bs-decode"
    }
  }
}
```

Note that it's detecting the branch as `bs-decode`, which isn't a real branch, and it's producing an incorrect `service` URL as a result.
